### DISCUSSION TITLE
Is It possible to change the Technical name

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#10968] [Q&A]
I need to change the technical name because it need to correct it.

![image](https://user-images.githubusercontent.com/5202291/148281236-cb042585-c84f-45d3-bfcd-35689ebc504a.png)

**Comment by ns8482e** [comment]
Not allowed in UI.  

However you can change it from  Content Definition - Do only If you know what you are doing and aware of consequences of such change.

**Comment by Skrypt** [comment]
It is equivalent to changing a database table name which requires some data migration done with SQL.

**Comment by saleemhaq** [comment]
Can you share with me the whole procedure please from start to finish?

**Comment by ns8482e** [comment]
@saleemhaq  It seems you misunderstood @Skrypt response - he referred database table rename as metaphor.

There is no such procedure - It's discouraged to rename technical name and that's why it's not allowed in UI.

However If you really want to rename technical name - you can  - by changing in Content Definition that will apply to all new content items, However existing content items will be broken - and literally you need to go through `json` of all content items that uses that old part name and update.
